examples/static_meshlets/shaders/meshlet.task

```diff
@@ -9,79 +9,27 @@
 #include "cpu_gpu_shared_config.h"
 #include "glsl_helpers.glsl"
 
+// 32 executions in parallel
 layout(local_size_x = 32, local_size_y = 1, local_size_z = 1) in;
 
-layout(set = 0, binding = 1) uniform CameraTransform
-{
-	mat4 mViewProjMatrix;
-} ubo;
-
-//-------------------------------------
-#if !USE_REDIRECTED_GPU_DATA
-struct meshlet
-{
-	uint vertices[64];
-	uint8_t indices[378];
-	uint8_t vertex_count;
-	uint8_t triangle_count;
-};
-#else
-struct meshlet
-{
-	uint data_offset;
-	uint8_t vertex_count;
-	uint8_t triangle_count;
-};
-#endif
-
-struct extended_meshlet
-{
-	mat4 mTransformationMatrix;
-	uint mMaterialIndex;
-	uint mTexelBufferIndex;
-	
-	meshlet mGeometry;
-};
-
-layout(set = 3, binding = 0) uniform  samplerBuffer positionBuffers[];
-layout(set = 3, binding = 1) uniform usamplerBuffer indexBuffers[];
-layout(set = 3, binding = 2) uniform  samplerBuffer normalBuffers[];
-layout(set = 3, binding = 3) uniform  samplerBuffer texCoordsBuffers[];
-#if USE_REDIRECTED_GPU_DATA
-layout(set = 3, binding = 4) uniform usamplerBuffer meshletVertexIndices[];
-#endif
-layout(set = 4, binding = 0) buffer MeshletsBuffer { extended_meshlet values[]; } meshletsBuffer;
-//-------------------------------------
-
-// Color table for drawing each meshlet with a different color.
-//
-#define MAX_COLORS 10
-vec3 meshletcolors[MAX_COLORS] = {
-  vec3(1,0,0), 
-  vec3(0,1,0),
-  vec3(0,0,1),
-  vec3(1,1,0),
-  vec3(1,0,1),
-  vec3(0,1,1),
-  vec3(1,0.5,0),
-  vec3(0.5,1,0),
-  vec3(0,0.5,1),
-  vec3(1,1,1)
-};
-
 out taskNV block
 {
 	uint meshletIndices[32];
 };
 
 void main() 
 {
+	// calculate and set the meshlet index for this execution
 	uint meshlet_group_id = gl_WorkGroupID.x;
 	uint thread_id = gl_LocalInvocationID.x;
 	uint mshIdx = meshlet_group_id * 32 + thread_id;
 
 	meshletIndices[thread_id] = mshIdx;
 
+	// Here you could do some culling on a per meshlet basis and only execute the meshlet shader for the surviving ones
```

Comment:
```suggestion
// Here culling on a per meshlet basis can be performed, so that the subsequent mesh shader is only executed for the surviving meshlets.
``` 

I don't like "you" so much. ;)
Also, it is the "mesh shader", not the "meshlet shader" => please check if "mesh shaders" are always called like that.